tests/models/unit_cases_gas/source_sink/src/run_source_sink.py

```diff
@@ -19,20 +20,29 @@ class TargetDemandGoal(Goal):
 
     order = 2
 
-    def __init__(self, state, target):
+    def __init__(self, state: str, target: Timeseries):
         self.state = state
 
         self.target_min = target
         self.target_max = target
         self.function_range = (0.0, 2.0 * max(target.values))
         self.function_nominal = np.median(target.values)
 
-    def function(self, optimization_problem, ensemble_member):
+    def function(
+        self, optimization_problem: CollocatedIntegratedOptimizationProblem, ensemble_member: int
+    ):
         return optimization_problem.state(self.state)
 
 
 class _GoalsAndOptions:
     def path_goals(self):
+        """
+        Here we add a goal to meet the specified gas demand.
```

Comment:
Suggestion 3rd person " "Add a goal to ..."

Reply:
done